### PR TITLE
Fixes #5363: Clean up package source to use a double-space indentation c...

### PR DESCRIPTION
**Pull request by jooooooon**
...onvention - implementation

### CODE REVIEW

**Comment by peckpeck** [comment]
@jooooooon This PR is not mergeable anymore, could you rebase it ? Thanks

**Comment by jooooooon** [comment]
@peckpeck Updated

File: rudder-agent-thin/SOURCES/patches/rudder-agent-thin/0001-rudder-agent-to-rudder-agent-thin-debian.patch [hunk 1]
```diff
@@ -109,34 +109,34 @@ diff -Naur a/rules b/rules
 --- a/rules	2014-12-08 15:05:20.562188524 +0100
 +++ b/rules	2014-12-08 15:09:17.030590768 +0100
 @@ -15,8 +15,6 @@
- 	dh_testdir
- 	# dh_prep has been moved from 'install' target since we need to build LMDB first
- 	dh_prep
```
Comment (peckpeck) [inline rudder-agent-thin/SOURCES/patches/rudder-agent-thin/0001-rudder-agent-to-rudder-agent-thin-debian.patch]: You reindent a patch
This context is reindented, which means the original file has been reindented too, but I can't find where

File: rudder-agent-thin/SOURCES/patches/rudder-agent-thin/0001-rudder-agent-to-rudder-agent-thin-debian.patch [hunk 2]
```diff
@@ -109,34 +109,34 @@ diff -Naur a/rules b/rules
...
+   dh_testdir
+   # dh_prep has been moved from 'install' target since we need to build LMDB first
+   dh_prep
 -	# Add here commands to configure the package.
 -	cd SOURCES && ./perl-prepare.sh $(CURDIR)/SOURCES/fusioninventory-agent
```
Comment (peckpeck) [inline rudder-agent-thin/SOURCES/patches/rudder-agent-thin/0001-rudder-agent-to-rudder-agent-thin-debian.patch]: These patched lines are not reindented which seems ton contradict the fact that the original file has been reindented

**Comment by peckpeck** [comment]
Except for the patch files, everything's ok for me

**Pull request closed without merging**